sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -167,4 +170,22 @@ export class InstitutionController extends BaseController {
   async getUserTypesAndRoles(): Promise<InstitutionUserTypeAndRoleResponseDto> {
     return this.institutionService.getUserTypesAndRoles();
   }
+
+  @Delete("/user/:id")
+  async removeUserAuth(
+    @Param("id") id: string,
+    @UserToken() token: IUserToken,
+  ) {
+    // Get auth
+    const institutionUser = await this.institutionService.getUser(+id);
```

Comment:
Is this method ensuring that the user being deleted belongs to the current institution?

Reply:
👍